css-grid-classes/style.css

```diff
@@ -0,0 +1,45 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+h2 {
+  height: 15%;
+}
+
+.column-link-yoshi {
```

Comment:
This grid class name should be changed to reflect that it is for a column that is meant to be a third. Also, there is no measurement here to show that this is meant to be a third of the available area. What measurement would be appropriate for a third? Responsive, remember!